client/src/search/SubjectWrappedTypeahead.js

```diff
@@ -0,0 +1,3 @@
+import React from "react";
+
+class SubjectWrappedTypeahead extends React.Component {}
```

Comment:
Haha, I see you're continuing the typeahead cleanup (oops, delete)

Reply:
whoops 😂 thanks